requirements.txt

```diff
@@ -1,3 +1,4 @@
-Django==2.1
+Django==2.1.2
 feedparser==5.2.1
 pytz==2018.5
+## The following requirements were added by pip freeze:
```

Comment:
the whole `requirements-to-freeze.txt` thing is from this guy: https://www.kennethreitz.org/essays/a-better-pip-workflow

basically put unpinned top level dependencies in `requirements-to-freeze.txt` (to make upgrades a lot easier), and use `requirements.txt` as an explicit and exhaustive lockfile.